src/site/stages/build/page-builder/schemas/page.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable camelcase */
```

Comment:
Interesting! What is a `page` entity? I don't think I included it in [the docs](https://github.com/department-of-veterans-affairs/vets-website/tree/kelsonic-2791-tome-sync-testing-scaffolding/src/site/stages/build/page-builder/tests/entities).

Reply:
It's a type of `node`.
```
❯ rg '"target_id": "page"' | wc -l
221
```